### PR TITLE
feat: add context_screen_owner_slug to toggleSavedSearch analytics payload

### PR DESCRIPTION
**Pull request by gkartalis**
The type of this PR is: **Enhancement**

This PR resolves [FX-3040]

### Description

Added `context_screen_owner_slug` to `toggleSavedSearch`

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

#nochangelog

[FX-3040]: https://artsyproduct.atlassian.net/browse/FX-3040

### CODE REVIEW

**Comment by gkartalis** [comment]
@dblandin By reading the description of the ticket you mentioned:
 _Missing this one attribute which will require some work around the relay/component organization._ 
 
 In my eyes it looked simpler to pass down as props the slug from the parent component. Would love some feedback though because I might be missing something 🤓

**Review by mdole — APPROVED** [review]
Nice! I think passing the slug as a prop is a reasonable solution

File: src/lib/Components/Artist/ArtistArtworks/SavedSearchBanner.tsx [hunk 1]
```diff
@@ -19,12 +19,20 @@ import { useTracking } from "react-tracking"
 interface SavedSearchBannerProps {
   me?: SavedSearchBanner_me | null
   artistId: string
+  slug: string
```
Comment (dblandin) [inline src/lib/Components/Artist/ArtistArtworks/SavedSearchBanner.tsx:22]: **non-blocking**: What do you think of also artist-prefixing this name as `artistSlug`?

**Review by dblandin — COMMENTED** [review]
> In my eyes it looked simpler to pass down as props the slug from the parent component. Would love some feedback though because I might be missing something

Yeah, passing down the slug as a prop looks totally fine to me 👍🏼 

Left one non-blocking comment but overall LGTM 💯

**Review by dblandin — APPROVED** [review]

File: src/lib/Components/Artist/ArtistArtworks/SavedSearchBanner.tsx (hunk 1 above)
Comment (gkartalis) [reply src/lib/Components/Artist/ArtistArtworks/SavedSearchBanner.tsx:22]: sounds reasonable, will do it 👍

**Comment by gkartalis** [comment]
#squashongreen

**Pull request merged**